doc/microsite/src/main/tut/sbt/00_sbt.md

```diff
@@ -263,6 +263,17 @@ lazy val myProjectBom = inRoot.as.module
 
 ```
 
+If you want to include BOMs of you project, you can use the following syntax:
+
+```scala
+libraryDependencies ++= Seq(
+  YourSdk.R.my_project_bom,     // Runtime artifact
```

Comment:
Make it consistent with the example given in lines above, e.g.:

```scala
MyProjectBOM.R.my_lib,     // Runtime dependency, same as `groupId %% "my-lib" % version`
MyProjectBOM.T.my_lib,     // Runtime dependency in for test scope, same as `groupId %% "my-lib" % version % Test`
```

add `import com.mycompany.myproject.MyProjectBOM`

And add an example `my_lib` project that `MyProjectBOM` exports

```scala
lazy val myLib = ...
```